src/redux/studio-comment-actions.js

```diff
@@ -3,7 +3,7 @@ const eachLimit = require('async/eachLimit');
 const api = require('../lib/api');
 const log = require('../lib/log');
 
-const COMMENT_LIMIT = 20;
+const COMMENT_LIMIT = 25;
```

Comment:
I'd probably leave this as 20 for now. If we were to change this, we'd need to also change https://github.com/LLK/scratch-www/blob/develop/src/redux/comments.js#L5 which also comes in to play i think, and then we have to change the project one also.